fe/fe-core/src/main/java/org/apache/doris/analysis/Analyzer.java

```diff
@@ -186,6 +187,10 @@ public long getAutoBroadcastJoinThreshold() {
         return globalState.autoBroadcastJoinThreshold;
     }
 
+    public void putDict(int slotId, ColumnDict dict) {
+        globalState.descTbl.putDict(slotId, dict);
```

Comment:
Use  map slotId to dictId instead of ColumnDict, and use a separate map of dictId to ColumnDict could reduce the rpc transmission, cause in some scenes, multiple slot  might have the same ColumnDict